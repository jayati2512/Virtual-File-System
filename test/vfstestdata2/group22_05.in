createvfs demo 2
createvfs demo 22
mountvfs demo
makedir /home/ bin
addfile /home/bin/a.txt 
unmountvfs demo
movefile /home/bin/a.txt /home/bin.txt
mountvfs demo
updatefile /home/bin/d.txt /usr/include/b.txt
unmountvfs demo1
unmountvfs demo
