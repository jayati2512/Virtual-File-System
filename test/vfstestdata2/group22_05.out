createvfs_SUCCESS
DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
makedir_SUCCESS
VFS_INSUFFICIENT_ARGUMENTS
unmountvfs SUCCESS
VFS_NOT_MOUNTED
mountvfs_SUCCESS
INTERNAL_FILE_NOT_FOUND
DATA_FILE_NOT_FOUND
unmountvfs_SUCCESS
